Tests/DatadogTests/Datadog/RUM/WebView/WebRUMEventConsumerTests.swift

```diff
@@ -65,7 +65,7 @@ class WebRUMEventConsumerTests: XCTestCase {
 
     func testWhenValidWebRUMEventPassedWithoutRUMContext_itPassesToWriter() throws {
```

Comment:
convention/ All tests for this (and Logs) consumer are named with _"itPassesToWriter"_, which isn't now true. It should now say that respective message is passed to Core Event Bus.